index.html

```diff
@@ -364,13 +364,13 @@
                     <table class="simple">
                         <thead><tr><th>Current state of device buttons</th><th><code>buttons</code></th></tr></thead>
                         <tbody>
-                            <tr><td><b>Mouse move with no buttons pressed</b></td><td>0</td></tr>
-                            <tr><td>Left Mouse,<br>Touch Contact,<br>Pen contact (with no modifier buttons pressed)</td><td>1</td></tr>
+                            <tr><td><b>Mouse moved with no buttons pressed</b>,<br> Pen moved while hovering with no buttons pressed</td><td>0</td></tr>
```

Comment:
> Pen _moved_ while hovering with no buttons pressed

Note that "moved" here includes the cases when the "footprint" of the pen is unmoved: e.g. a change in <code>twist</code>. I think the pen still must move physically to generate an event, so this is fine.

Do we need a note?